cmd/minishift/cmd/daemon/proxy.go

```diff
@@ -46,7 +46,7 @@ func init() {
 	servicesProxyCmd.Flags().IntVarP(&proxyServerPortFromFlag, proxyServerPortFlag, "p", 3128, "The server port.")
 	servicesProxyCmd.Flags().StringVarP(&proxyUpstreamAddrFromFlag, proxyUpstreamFlag, "u", "", "The upstream proxy address.")
 	servicesProxyCmd.Flags().BoolVarP(&proxyReEncryptFlag, "reencrypt", "r", false, "Re-encrypt traffic")
-	ServicesCmd.AddCommand(servicesProxyCmd)
+	DaemonCmd.AddCommand(servicesProxyCmd)
```

Comment:
`servicesProxyCmd`?

Reply:
Yikes!!... Fixing it! I thought i fixed the names...